packages/components/src/jetpack-mobile-promo-card/images/index.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.png' {
```

Comment:
```suggestion
declare module '*.png';
```

Would this shorthand-declared module be capable of resolving the importing error?

Reply:
Yes, it looks like this does the trick. Thanks!